src/api/spec/support/shared_examples/features/boostrap_flag_tables.rb

```diff
@@ -65,20 +75,23 @@ def css_locator_for(repository, architecture)
     query_attributes[:flag] = flag_type
 
     disable_flag_field_for(repository: 'All', architecture: 'All')
-    expect(project.flags.reload.where(query_attributes.merge(status: :disable))).to exist
+    expect(project.flags.reload.where(query_attributes).pluck(:status)).to contain_exactly('disable')
```

Comment:
why is reload needed? b:confused:

Reply:
because the project.flags are manipluated by selenium - and the rspec object doesn't know

Reply:
I can try to leave it out